addons/calendar/static/src/views/attendee_calendar/attendee_calendar_model.js

```diff
@@ -41,7 +51,8 @@ export class AttendeeCalendarModel extends CalendarModel {
      * @override
      */
     buildRawRecord(partialRecord, options = {}) {
-        const result = super.buildRawRecord(...arguments);
+        options['duration_hour'] = this.defaultDuration;
+        const result = super.buildRawRecord(partialRecord, options);
```

Comment:
```suggestion
        options['duration_hour'] = this.defaultDuration;
        const result = super.buildRawRecord(partialRecord, {...options, duration_hour: this.defaultDuration});
```
It s a better practice to clone your object and not edit it directly :)

Reply:
Indeed, thanks for your inputs !